### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by kseniyaLine**
created simple layout for all pages+body as an example

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
```
Comment (mtalstykh-exadel) [inline .idea/.gitignore]: you have some problems with .gitignore. ".Idea" should be ignored

File: .idea/LayOut.iml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/temp" />
+      <excludeFolder url="file://$MODULE_DIR$/.tmp" />
+      <excludeFolder url="file://$MODULE_DIR$/tmp" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (mtalstykh-exadel) [inline .idea/LayOut.iml]: same here

File: .idea/inspectionProfiles/Project_Default.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
+    <inspection_tool class="Eslint" enabled="true" level="WARNING" enabled_by_default="true" />
+    <inspection_tool class="Stylelint" enabled="true" level="ERROR" enabled_by_default="true" />
+  </profile>
+</component>
```
Comment (mtalstykh-exadel) [inline .idea/inspectionProfiles/Project_Default.xml]: same

File: .idea/modules.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/LayOut.iml" filepath="$PROJECT_DIR$/.idea/LayOut.iml" />
+    </modules>
+  </component>
+</project>
```
Comment (mtalstykh-exadel) [inline .idea/modules.xml]: same

File: .idea/vcs.xml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```
Comment (mtalstykh-exadel) [inline .idea/vcs.xml]: same

File: src/components/body/Body.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    <Fragment>
+      <div className="body"> {children}</div>
+    </Fragment>
+  );
+};
+
+Body.propTypes = {
+  children: PropTypes.object,
+};
+export default Body;
```
Comment (mtalstykh-exadel) [inline src/components/body/Body.js]: we do not need the body file

File: src/components/layout/Layout.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+import React from "react";
+import { Fragment } from "react";
+import Body from '../body/Body.js';
+
+const Layout = () => {
+  return (
+    <Fragment>
+      <main className="main">
+        {/* <Header/> */}
+        <Body>Hello it is body</Body>
```
Comment (mtalstykh-exadel) [inline src/components/layout/Layout.js]: let it be 
<>
{header}
<main>{children}</main>
</>

File: src/components/layout/Layout.js [hunk 8]
```diff
@@ -0,0 +1,19 @@
+import "./Layout.scss";
+import React from "react";
+import { Fragment } from "react";
```
Comment (mtalstykh-exadel) [inline src/components/layout/Layout.js]: use <> </> without importing Fragment

File: src/components/layout/Layout.scss [hunk 9]
```diff
@@ -0,0 +1,7 @@
+.main {
+  width: 100%;
+  height: 100%;
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
```
Comment (mtalstykh-exadel) [inline src/components/layout/Layout.scss]: remove the line couse we will connect reset.scss

File: src/components/layout/Layout.js [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+
+
+const Layout = ({children}) => {
+  return (
+    <Fragment>
+      <main className="main">
+        {/* <Header/> */}
```
Comment (mtalstykh-exadel) [inline src/components/layout/Layout.js]: almost) move it one level higher

File: src/components/layout/Layout.js [hunk 11]
```diff
@@ -0,0 +1,15 @@
+import "./Layout.scss";
+import React from "react";
+
+
+const Layout = ({children}) => {
+  return (
+    <div>
+      <main className="main"> {/* <Header/> */}{/*<{children}>*/}</main>
```
Comment (mtalstykh-exadel) [inline src/components/layout/Layout.js]: 🐱‍🚀

File: src/components/layout/Layout.scss [hunk 12]
```diff
@@ -0,0 +1,6 @@
+.main {
```
Comment (mtalstykh-exadel) [inline src/components/layout/Layout.scss:1]: 🐱‍🚀

File: src/components/layout/Layout.scss [hunk 13]
```diff
@@ -0,0 +1,6 @@
...
+  width: 1280px;
```
Comment (mtalstykh-exadel) [inline src/components/layout/Layout.scss]: max-width, please

File: src/components/layout/Layout.js [hunk 14]
```diff
@@ -0,0 +1,18 @@
+import "./Layout.scss";
+import React from "react";
+import {PropTypes} from "@material-ui/core";
+
+const Layout = ({children}) => {
+  return (
+     <>
```
Comment (mtalstykh-exadel) [inline src/components/layout/Layout.js]: format the code

**Review by mtalstykh-exadel — APPROVED** [review]

**Review by tensenrey — APPROVED** [review]

**Pull request merged**